packages/grid/src/vaadin-grid-sorter.js

```diff
@@ -202,7 +208,13 @@ class GridSorter extends ThemableMixin(DirMixin(PolymerElement)) {
       return;
     }
 
-    this.dispatchEvent(new CustomEvent('sorter-changed', { bubbles: true, composed: true }));
+    this.dispatchEvent(
+      new CustomEvent('sorter-changed', {
```

Comment:
Let's also change types for `sorter-changed` event and add `GridSorterChangedEvent` with typed `detail`:

https://github.com/vaadin/web-components/blob/5004363edf7456af0703d3dc1e75593ca4f76b99/packages/grid/src/vaadin-grid-sorter.d.ts#L17

Something like this:

```ts
/**
 * Fired when the `path` or `direction` property changes.
 */
export type GridSorterChangedEvent = CustomEvent<{ shiftClick: boolean }>;
```

Reply:
Done.